java/integTest/src/test/java/glide/ErrorHandlingTests.java

```diff
@@ -12,10 +12,11 @@
 import glide.api.models.exceptions.RequestException;
 import java.net.ServerSocket;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
 import lombok.SneakyThrows;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
 
+@Timeout(20)
```

Comment:
nit: looks like all the tests previously had a timeout of 10 seconds, should we change this value to 10? Same question for the other files

Reply:
Fixed